modules/nw-networkpolicy-about.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// networking/configuring-networkpolicy.adoc
+
+[id="nw-networkpolicy-about-{context}"]
+
+= About network policy
+
+In {product-title} {product-version} `NetworkPolicy` is the default mode for the
+`OpenShiftSDN` plug-in for the Kubernetes Container Network Interface (CNI).
+
+[NOTE]
+====
+The `v1` NetworkPolicy features are available only in {product-title}. This
+means that egress policy types, IPBlock, and combining `podSelector` and
+`namespaceSelector` are not available in {product-title}.
+====
+
+In a cluster network isolation is controlled entirely by
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network/network-policy.md[`NetworkPolicy`
+objects]. By default, all pods in a project are accessible from other pods and
+network endpoints. To isolate one or more pods in a project, you can create
+`NetworkPolicy` objects in that project to indicate the allowed incoming
+connections. Project administrators can create and delete `NetworkPolicy`
+objects within their own project.
+
+Pods that do not have `NetworkPolicy` objects pointing to them are fully
```

Comment:
I would replace "pointing to" with something like "selecting".

Reply:
Perhaps this instead?

> If a Pod is selected by one or more `NetworkPolicy` objects, then the Pod will
> accept only connections that are allowed by at least one of those `NetworkPolicy`
> objects. A Pod that is not selected by any `NetworkPolicy` objects is fully
> accessible.